meta/innersource-spelling.md

```diff
@@ -24,15 +24,15 @@ For new patterns we stick to the spelling **InnerSource**. Old patterns will be
 
 ## It can be hard to use the term "InnerSource" in a sentence
 
-Some people have had a hard time with the camel-case term **InnerSource** because they’re not sure how to use it in a sentence grammatically.
+Some people have had a hard time with the Pascal case term **InnerSource** because they’re not sure how to use it in a sentence grammatically.
 
 We recommend not to turn the term into a verb, but rather using it as a noun or adjective.
 
 Recommended use:
 
-“We use InnerSource in that repo”
+“We published the repository as InnerSource.”
 or
-“We use InnerSource methods in that repo”
+“We published the repository under an InnerSource license.”
```

Comment:
```suggestion
“We use InnerSource methods in that repo.”
```

I would prefer to keep the original example here, as using InnerSources isn't that common (from what I know), so I don't want to suggest that they are.

However if you want to add any other example besides licenses, please do. Doesn't hurt to have 3-4 or even more examples of recommended use.